tests/webhook_create_mock_test.go

```diff
@@ -0,0 +1,177 @@
+package tests
+
+import (
+    "context"
+    "fmt"
+    "net/http"
+    "testing"
+
+    "github.com/fivetran/go-fivetran"
+    "github.com/fivetran/go-fivetran/tests/mock"
+)
+
+const (
+    WEBHOOK_URL         = "https://webhook.site/abe96072-249c-40bc-a12d-8b92750175e2"           // Unstable test url
```

Comment:
let's just use WEBHOOK_URL = "webhook_url" - the value should be just string, we return the same value in mocked response - so it's better to avoid using any kind of "real" urls in tests.